client/public/loginPage/loginScript.js

```diff
@@ -0,0 +1,45 @@
+const loginForm = document.getElementById("loginFrm");
+const emailInpt = document.getElementById("emailInpt");
+const passwordInpt = document.getElementById("passwordInpt");
+
+loginForm.addEventListener("submit", async (e) => {
+  e.preventDefault();
+  const userInfo = {
+    email: loginForm.email.value,
+    password: loginForm.password.value,
+  };
+  try {
+    if (userInfo.email !== null || userInfo.password !== null) {
+      const data = await login(userInfo);
+      if (data.isLoggedIn) {
+        if (data.role === "manager") {
+          window.location.replace("http://localhost:4000/public/dashboard");
+        }
+        if (data.role === "employee") {
+          window.location.replace("http://localhost:4000/public/dashboard");
+        }
+      } else {
+        errorMsg.style.display = "block";
+      }
+    }
+  } catch (error) {}
+});
+
+emailInpt.addEventListener("focus", () => {
+  errorMsg.style.display = "none";
+});
+passwordInpt.addEventListener("focus", () => {
+  errorMsg.style.display = "none";
+});
+
+const login = async (userInfo) => {
+  const response = await fetch("http://localhost:4000/api/auth/login", {
```

Comment:
consider moving the API calls into an API service on the client side. for example:
```
// apiService.js
const apiService  = {
 auth: {
   login: id => { /* do fetch stuff here */,
   logout: () => {...},
  },
  ...,
}
```